webpack.config.js

```diff
@@ -27,7 +27,11 @@ module.exports = {
       {
         test: /\.css$/,
         use: ["style-loader", "css-loader"]
-      }
+      },
+      {
+        test: /\.(png|jpe?g|gif)$/i,
+        type: 'asset/resource'
```

Comment:
allows webpack to load images - can also use file-loader but not recommended bc its being deprecated